CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## X.X.X (YYYY-MM-DD)
+
+-   Inline named object instances are now available when using completion lists, hover and other features like object instances are.
```

Comment:
Lots of changes, so take this with a grain of salt.

```suggestion
-   Named inline object instances now appear (like “normal” instances already do) in lists provided for autocompletion, via hover, etc.
```

Reply:
I may need to better understand what those instances are to improve the suggestion.

Reply:
(cf. comment in `YmlParser.g4`)